pyscriptjs/examples/fetch_data.py

```diff
@@ -0,0 +1,37 @@
+from pydoc import doc
+from js import console, fetch, JSON , document
```

Comment:
I think we should encourage users to use more of the Python ecosystem when possible. In this case, I'd propose to use pyfetch instead of fetch. So `from pyodide.http import pyfetch`